### DISCUSSION TITLE
difft not working as an external diff

### DISCUSSION THREAD

**Discussion by shauryagoel** [jesseduffield/lazygit#3267] [Q&A] [answered]
I am trying to use difft as mentioned [here](https://github.com/jesseduffield/lazygit/blob/master/docs/Custom_Pagers.md#using-external-diff-commands). However, there is no effect when doing so as unstaged changes  diff is same. Here is my config- 
```
git:
  paging:
    externalDiffCommand: difft --color=always
```
MacOS: 14.1.2
Lazygit version: 0.40.2
Difftastic version: 0.53.1
Git version: 2.39.3

**Comment by Pencilcaseman** [comment]
I have the same issue using an identical config. It seems like it's not using `externalDiffCommand` as it works for other tools like `delta`.

```
build source=nix
version=0.40.2
os=darwin
arch=arm64
git version=2.43.0
```

**Comment by stefanhaller** [comment, marked as answer]
Copied from https://github.com/jesseduffield/lazygit/issues/3304#issuecomment-1925863102: `externalDiffCommand` was only added after 0.40.2 was released. You'll have to use a master build to use it, or wait for the next release.

**Reply by Pencilcaseman** [reply]
Yes, my bad. I found that issue after some more research but didn't have time to update my comment. Cheers! That fixes it, and it works perfectly.
